pkg/util/utils.go

```diff
@@ -69,6 +70,50 @@ func StringInSlice(s string, sl []string) bool {
 	return false
 }
 
+// ParseChanges returns key, value(s) pair for given option.
+func ParseChanges(option string) (key string, vals []string, err error) {
+	// Supported format as below
+	// 1. key=value
+	// 2. key value
+	// 3. key ["value","value1"]
+	if strings.Contains(option, " ") {
+		// This handles 2 & 3 conditions.
+		var val string
+		tokens := strings.SplitAfterN(option, " ", 2)
+		if len(tokens) < 2 {
+			return "", []string{}, fmt.Errorf("invalid key value %s", option)
+		}
+		key = strings.Trim(tokens[0], " ") // Need to trim whitespace part of delimeter.
+		val = tokens[1]
+		if strings.Contains(tokens[1], "[") && strings.Contains(tokens[1], "]") {
+			//Trim '[',']' if exist.
+			val = strings.TrimLeft(strings.TrimRight(tokens[1], "]"), "[")
+		}
+		vals = strings.Split(val, ",")
+	} else if strings.Contains(option, "=") {
+		// handles condition 1.
+		tokens := strings.Split(option, "=")
+		key = tokens[0]
+		vals = tokens[1:]
+	} else {
+		// either ` ` or `=` must be provided after command
+		return "", []string{}, fmt.Errorf("invalid format %s", option)
+	}
```

Comment:
What happens if someone puts in `--change LABEL = blue=image`  ?

Reply:
with current logic, it will be an error case. Though we can remove whitespaces in this condition.
Please let me know WDYT.

Reply:
error, condition works for me @kunalkushwaha , thanks!